app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/overview.scala.html

```diff
@@ -0,0 +1,49 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.incometaxsubscriptioneligibilityfrontend.config.AppConfig
+@import views.html.helper.form
+@import uk.gov.hmrc.incometaxsubscriptioneligibilityfrontend.views.html.templates.main_template
+
+
+@(postAction: Call)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@main_template(
+    title = messages("overview.title"),
+    mainClass = Some("full-width")
+) {
+
+    <div class="grid-row interrupt-card">
+        <div class="column-two-thirds">
+
+            <h1>@messages("overview.heading")</h1>
+
+            <div class="form-group">
+                <p>@messages("overview.line_1")</p>
+                <p>@messages("overview.line_2")</p>
+                @Html(messages("overview.link_text", appConfig.guidanceUrl))
```

Comment:
```suggestion
                <a id="guidance" href=@appConfig.guidanceUrl>@messages("overview.link_text")</a>
```
Can the link tag be taken out of the message?